### DISCUSSION TITLE
Blogging functions

### DISCUSSION THREAD

**Discussion by sergefdrv** [colinwilson/lotusdocs#41] [Q&A]
Would it make sense to use this template if I also need to publish blog posts?

**Comment by colinwilson** [comment]
The theme is documentation focused.

That's not to say that it won't function as a blog theme, but there a other themes better suited to blogging than Lotus Docs.

Have you checked out [https://themes.gohugo.io/](https://themes.gohugo.io/) ? A lot of great themes on there for blogging.

**Reply by sergefdrv** [reply]
Thanks for responding. I would like to have both docs and blog in my project site. Now, I'm looking into [Docusaurus](https://docusaurus.io/), which seems to have all the features I need.
